modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -279,10 +279,7 @@ def create_actors(num_actors):
 
 
 def _split_data_across_actors(
-    actors: List,
-    set_func,
-    X_parts,
-    y_parts,
+    actors: List, set_func, X_parts, y_parts, **dmatrix_params
```

Comment:
```suggestion
    actors: List, set_func, X_parts, y_parts
```

`dmatrix_params` is already presented in `set_func`

Reply:
this was changed